Kitodo/src/main/java/org/kitodo/config/enums/ParameterCore.java

```diff
@@ -618,8 +618,10 @@ public enum ParameterCore implements ParameterInterface {
     /*
      * Elasticsearch properties
      */
-    ELASTICSEARCH_BATCH(new Parameter<>("elasticsearch.batch", 500)),
     ELASTICSEARCH_INDEXLIMIT(new Parameter<>("elasticsearch.indexLimit", 5000)),
+    ELASTICSEARCH_BATCH(new Parameter<>("elasticsearch.batch", 500)),
+    ELASTICSEARCH_ATTEMPTS(new Parameter<>("elasticsearch.attempts", 10)),
+    ELASTICSEARCH_THREADS(new Parameter<>("elasticsearch.threads", 4)),
```

Comment:
Please add this new configuration options to the proper section into main `kitodo_config.property` file with a good documentation.

Reply:
Done, see below.